backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Get_QtyPattern.sql

```diff
@@ -0,0 +1,43 @@
+/*
+ * #%L
+ * de.metas.fresh.base
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+DROP FUNCTION IF EXISTS report.Get_QtyPattern(numeric)
+;
+
+CREATE FUNCTION report.Get_QtyPattern(UOM_StdPrecision numeric) RETURNS text
+    LANGUAGE plpgsql
+AS
```

Comment:
pls rename this to ```report.getQtyPattern```
Try to follow the example naming that we have.

Reply:
done